webpack/assets/javascripts/react_app/components/toastNotifications/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Toast from './toastListitem/';
+import * as ToastActions from '../../redux/actions/toasts';
+import { connect } from 'react-redux';
+import './ToastList.scss';
+
+class ToastsList extends Component {
+
+  render() {
+    const { messages, hideToast } = this.props;
+
+    const visibleToasts = Object.keys(messages).reduce((curr, next) => {
```

Comment:
Why not something like `Object.values(messages).find( (val) => (val.visible))`? It would also simplify the map below for returning the visible notifications.